### PR TITLE
Add Karsha Exchange

### PR DESCRIPTION
**Pull request by khashmeshab**

### CODE REVIEW

**Comment by zquestz** [comment]
I see you have been around for a while, but I can't seem to find any user feedback or reviews for your site. Generally we want to guarantee the quality of anything we link on bitcoincash.org. Can you point us to user reviews and testimonials (forums, reddit, etc)?

**Comment by khashmeshab** [comment]
TL;DR: No, there isn't any review of us because we worked local, we were mostly e-currency exchange, we were mostly inactive for the past few years. But it would be great if you add us to the list for a test run. You won't regret it. Thank you.

Long Version:

Hello. Thank you for your reply.

Well, we were mostly an e-currency exchange website, and worked locally. So, most of our customers were local. The few international customers wrote some positive reviews in some forums, which I think those e-currency related forums are closed now! I know for sure that there was a good official review of our website at e-Commerce Journal. But I think because we were mostly inactive for the previous years, those few reviews are either deleted or got lost in history. We were also the official representative and/or certified exchange of: Perfect Money, OKPAY, and EgoPay, for years. We were the main exchange of BTC-e to exchange their EgoPay. But our inactivity made us disappear from the market.

Now we've got new investors from Russia, China, UK, and Canada, and started fresh and all over again. Our altcoin exchange system is only one months old. We have implemented a smart system which dynamically adjust the price and spread of cryptocurrencies in a way that we could both provide the best buy/sell price to our customers, and give as much profit as possible to our investors. We may open up our investment system publicly in the feature, so everyone could invest and automatically trade with their cryptocurrencies. No need to register/take offers anymore!

If it helps, please take a look at here:
https://en.bitcoin.it/wiki/Karsha_Mobile_Payment_Interface
And here:
https://en.bitcoin.it/wiki/Karsha_Shopping_Cart_Interface

These systems are now defunct. But it shows our history a little more. You may see some snapshots at web.archive.org too.

Thank you for your time. I would be very thankful if you could give us a chance. But I would totally understand if you'd need a well-reviewed site in your list.

**Comment by fyookball** [comment]
the lack of reviews/history is a concern.

**Comment by khashmeshab** [comment]
Well, is the glass half empty or half full?!

Working for 8 years, without a single positive review,

Or,

Working for 8 years, without a single negative review?!

**Comment by fyookball** [comment]
good point :)

**Comment by zquestz** [comment]
I will post the update later tonight. Best of luck on growing your exchange!

**Comment by khashmeshab** [comment]
Thank you very much. You won't regret your decision. We won't let you down.

**Comment by zquestz** [comment]
You are posted!

**Comment by khashmeshab** [comment]
Thank you.
